Documentation/trying-out-rkt.md

```diff
@@ -16,6 +16,8 @@ cd rkt-v1.8.0
 ./rkt help
 ```
 
+Note that in order to fully utilize rkt there is a couple of additional steps to be performed. Therefore we highly recommend installing rkt using your local package manager. If rkt is not available for your Linux distribution please consider to try rkt using Vagrant as described below.
```

Comment:
"Like most applications, the easiest way to run rkt is to install it with your system's package manager, like apt on Debian or dnf on Fedora. If your distribution doesn't include a rkt package, or your operating system isn't Linux at all, running rkt in a Vagrant virtual machine can be nearly as simple. The instructions below start a virtual machine with rkt installed and ready to run."